cpp/src/slice2swift/Gen.cpp

```diff
@@ -993,7 +993,7 @@ Gen::TypesVisitor::visitEnum(const EnumPtr& p)
     out << nl << "/// Read an enumerated value.";
     out << nl << "///";
     out << nl << "/// - Returns:  The enumerated value.";
-    out << nl << "func read() throws -> sending " << name;
+    out << nl << "func read() throws -> " << name;
```

Comment:
Was this sending bogus?

Reply:
It was not bogus, just unnecessary. We only need sending for NonSendable types, and all generated enums are now Sendable.